utility/system.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+def is_path_empty(path):
```

Comment:
I know it's pretty self explanatory but we should still have some pydoc for this function.